deploy/iso/minikube-iso/package/automount/minikube-automount

```diff
@@ -117,6 +121,10 @@ if [ -n "$BOOT2DOCKER_DATA" ]; then
     mkdir /var/lib/kubelet
     mount --bind /mnt/$PARTNAME/var/lib/kubelet /var/lib/kubelet
 
+    mkdir -p /mnt/$PARTNAME/var/lib/kubelet/pods
```

Comment:
I don't think adding /var/lib/kubelet adds value to what you are trying to fix. I'd just remove it.

Reply:
done